### PR TITLE
Update vision-transformer-for-objection-detection.mdx

### PR DESCRIPTION
**Pull request by mohammad-gh009**
replace "The Need to Fine-tune Models in Object Detection"  with "Why do we need fine-tuning for object detection"

### CODE REVIEW

File: chapters/en/unit3/vision-transformers/vision-transformer-for-objection-detection.mdx [hunk 1]
```diff
@@ -27,7 +27,7 @@ There are a lot of of applications around object detection. One of the most sign
 
 To deepen your understanding of the ins-and-outs of object detection, check out our [dedicated chapter](/chapters/en/Unit%206%20-%20Basic%20CV%20Tasks/object_detection.mdx) on Object Detection 🤗
 
-### The Need to Fine-tune Models in Object Detection 🤔
+### Why do we need fine-tuning for object detection? 🤔
```
Comment (merveenoyan) [inline chapters/en/unit3/vision-transformers/vision-transformer-for-objection-detection.mdx:30]: We particularly asked the author not to make this a question

**Review by bellabf — COMMENTED** [review]
HI @mohammad-gh009, I was requested to re-review, but there are no new commits. Could you address @merveenoyan comments?

File: chapters/en/unit3/vision-transformers/vision-transformer-for-objection-detection.mdx (hunk 1 above)
Comment (johko) [reply chapters/en/unit3/vision-transformers/vision-transformer-for-objection-detection.mdx:30]: Late reply from my side, but I think this comment still needs attention @mohammad-gh009

**Comment by johko** [comment]
Hey @mohammad-gh009, do you have time to do the small change that was requested? Else I will will close this PR for now without merging, so that we don't have to many Zombie PRs hanging around

**Pull request closed without merging**